index.html

```diff
@@ -65,15 +65,194 @@ <h1 class="intro-heading">
+							<li>Ruby on Rails</li>
+							<li>CSS</li>
+							<li>Javascript</li>
+							<li>HTML</li>
+						</ul>
+						<div class="button-container">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+					<div class="card">
+						<img src="/images/Img-Placeholder.png" alt="" />
+						<div class="title-container">
+							<h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+						</div>
+						<ul class="tech">
+							<li>Ruby on Rails</li>
+							<li>CSS</li>
+							<li>Javascript</li>
+							<li>HTML</li>
+						</ul>
+						<div class="button-container">
+							<button type="button" class="project-button">See Project</button>
+						</div>
+					</div>
+				</div>
+			</section>
+			<section class="about-me">
+				<div class="about-me__text">
+					<img class="side-image" src="/images/Mask Group.svg" alt="" />
+					<h2 class="about-me__header">About me</h2>
+					<p class="about-me__paragraph">
+						I can help you build a product,feature or website Look through some
+						of my work and experience! If you like what you see and have a
+						project you need coded, don't hestiate to contact me.
+					</p>
+					<button type="button" class="about-me__button">Get My Resume</button>
+				</div>
+				<div class="about-me__cards">
+					<div class="about-me__card">
+						<div class="about-me__image">
+							<img
+								class="icon-language"
+								src="/images/icon-languages.svg"
+								alt="" />
+						</div>
+						<div class="about-me_cards__title">
+							<h3>Languages</h3>
+						</div>
+						<ul class="language-lists">
+							<li class="language-list">Javascript</li>
+							<li class="language-list">Ruby</li>
+							<li class="language-list">HTML</li>
+							<li class="language-list">CSS</li>
+						</ul>
+					</div>
+					<div class="about-me__card">
+						<div class="about-me__image">
+							<img
+								class="icon-language"
+								src="/images/icon-frameworks.svg"
+								alt="" />
+						</div>
+						<div class="about-me_cards__title">
+							<h3>Framework</h3>
+						</div>
+						<ul class="language-lists">
+							<li class="language-list">React.js</li>
+							<li class="language-list">Ruby on Rails</li>
+							<li class="language-list">Rspec</li>
+							<li class="language-list">Selenium</li>
+							<li class="language-list">Capybara</li>
+						</ul>
+					</div>
+					<div class="about-me__card">
+						<div class="about-me__image">
+							<img class="icon-language" src="/images/icon-skills.svg" alt="" />
+						</div>
+						<div class="about-me_cards__title">
+							<h3>Skills</h3>
+						</div>
+						<ul class="language-lists">
+							<li class="language-list">Database Management</li>
+							<li class="language-list">Version Control</li>
+							<li class="language-list">CLI</li>
+							<li class="language-list">Web Development</li>
+							<li class="language-list">API Design</li>
+						</ul>
+					</div>
```

Comment:
- [ ] [OPTIONAL] The `background-image` and the `images` of your skills section are not showing, and this isn't really good for user experience. It could be better if enhanced and checked. Kindly look into it if you can, but it will be really okay for your page :hammer_and_wrench: